### PR TITLE
fix(ingest): bigquery - missing sqlalchemy dep and row count fix

### PR DESCRIPTION
**Pull request by treff7es**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py [hunk 1]
```diff
@@ -349,8 +349,8 @@ def get_tables_for_dataset(
                 )
                 if "last_altered" in table
                 else None,
-                size_in_bytes=table.bytes if "bytes" in table else None,
-                rows_count=table.row_count if "row_count" in table else None,
+                size_in_bytes=table.bytes if "bytes" in table.keys() else None,
```
Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py:352]: why do we need .keys()?

Comment (jjoyce0510) [inline metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py:352]: Can table be None?

Comment (treff7es) [reply metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py]: table can't be null as this is return bigquery Row objects from the RowIterator

**Review by jjoyce0510 — APPROVED** [review]
LGTM. Thanks. Are there tests for this?

**Pull request merged**